cf_remote/spawn.py

```diff
@@ -317,7 +317,10 @@ def spawn_vm_in_aws(
     else:
         if any(vm.state in (0, "running") and vm.name == name for vm in existing_vms):
             raise ValueError("VM with the name '%s' already exists" % name)
-
+    if not platform in aws_platforms:
+        print("Platform '%s' does not exist.\nList of available platforms:\n%s" % (platform,
+                         "\n".join(cloud_data.aws_platforms.keys())))
+        return None
```

Comment:
I did not run your code, but the error handling looks very different from what is on the line above;

```suggestion
    if not platform in aws_platforms:
        raise ValueError(
            "Platform '%s' does not exist.\nList of available platforms:\n%s"
            % (platform, "\n".join(cloud_data.aws_platforms.keys())))
```

Which may or may not be important. (Will the error messages look the same? Is there some special handling when catching the ValueErrors? Will the exit code be the same?)

So, either update them to be the same, or explain in the commit message how and why they are different.